src/components/scenes/GuiPluginListScene.tsx

```diff
@@ -344,10 +345,17 @@ class GuiPluginList extends React.PureComponent<Props, State> {
     }
   }
 
-  handleError = (error: any): void => {
+  handleError = (error: unknown): void => {
     const { countryCode, stateProvinceCode } = this.props
 
-    if (error[0].quoteError.errorType === 'regionRestricted') {
+    const regionError = asMaybe(
+      asArray(
+        asObject({
+          quoteError: asObject({ errorType: asValue('regionRestricted') })
+        })
+      )
+    )(error)
+    if (regionError != null && regionError.length > 0 && regionError?.[0].quoteError.errorType === 'regionRestricted') {
```

Comment:
This can be simplified to `if (regionError != null && regionError.length > 0)`, since the cleaner checks for the 'regionRestricted' value already.